api/src/main/resources/application.properties

```
@@ -3,4 +3,8 @@ spring.datasource.username=root
 spring.datasource.password=password
 spring.datasource.driverClassName=com.mysql.cj.jdbc.Driver
 spring.jpa.database=MYSQL
-spring.jpa.hibernate.ddl-auto=update
\ No newline at end of file
+spring.jpa.hibernate.ddl-auto=update
```

```suggestion
spring.jpa.hibernate.ddl-auto=none
```
アプリ起動時にJPAの機構でDDLが走っちゃうの止めておきましょう。
マイグレーションはflywayから行う